refs/schemas/priv/RFC-PRIV.md

```diff
@@ -0,0 +1,887 @@
+
+### Privacy Request Response
+
+Systems SHOULD respond to [Privacy Requests](#privacy-request).
+Regardless of the [scenario (Responding to the Data Subject directly or to the System)](./scenarios.md#response) Systems SHOULD generate a response using the Privacy Request Interchange Vocabulary.
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `response-id` | 1 | Unique ID for referring to this request in the [uuid](https://www.rfc-editor.org/rfc/rfc4122.html) format |
+| `in-response-to` | 1 | `request-id` of the Privacy Request to which response is made or `demand-id` of the particular Demand to which response is made, in the [uuid](https://www.rfc-editor.org/rfc/rfc4122.html) format |
+| `date` | 1 | Date and Time when Privacy Request Response was created in JSON Schema [date-time](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.1) format |
+| `system` | 1 | System ID of the System having generated the response. A String in the format of URI according to [RFC3986 of IETF](https://www.rfc-editor.org/rfc/rfc3986) |
+| `requested-action` | 0-1 | Optional information about the action that was demanded, and to which the response is made. [Action](#actions)|
+| `data-subject` |  0-* | Optional indication of the [Data Subject Identities](#decentralized-identity-of-data-subjects) to which the response refers to |
+| `status` | 1 | [Status Terms](#statuses) |
+| `motive` | 0-* | [Motive Terms](#motives) |
+| `answers` | 0-* | [Terms](#terms) |
+| `message` | 0-1 | Optional string comment, motivation or explanation of Demand |
+| `lang` | 0-1 | Optional string Language of textual message associated with demands in the format of [FRC5646](https://datatracker.ietf.org/doc/rfc5646/) |
+| `includes` | 0-* | Optionally an array of one or more [Privacy Request Response](#privacy-request-response)s |
+| `data` | 0-* | Optionally concrete data to which access is being given|
+
+A Privacy Request Response MUST have:
+- a unique ID,
+- a clear indication of the particular Privacy Request or the particular Demand to which its content refers,
+- a date,
+- a status.
+
+Privacy Request Responses CAN be nested. One can imagine a Privacy Request Response to a particular Privacy Request, that `includes` Privacy Request Responses to the particular Demands made in that Privacy Request. Several Systems MAY respond to the same Privacy Request or Demand, and one System MAY nest them in order to gather them and send them back to the Data Subject (in the [Coordinated Response secenario](./scenarios.md#coordinated-response)).
+
+When a Demand is being denied, the Privacy Request Response MUST provide a `motive`.
+
+>To illustrate the `answers` value, we can imagine a `TRANSPARENCY.DATA-CATEGORIES` Demand, to which a response may include `answers`: `CONTACT`, `IMAGE`. Or a `TRANSPARENCY.KNOWN` Demand to which the answer may include `answers`: `YES` from [Boolean Terms](#boolean).
```

Comment:
There is no `answer`. Just `data` which is a free form of data about how to contact DPO, and who the DPO is. It can be a text or structured data, and should be configured in properties as specified under [Storing General Information](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md#storing-general-information)

Reply:
@m4rk055 @filipblnt see also the [UROPA](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/priv/RFC-PRIV.md#uropa) section I just added